ajax/libs/vuedraggable/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "vuedraggable",
+  "version": "2.11.0",
+  "filename": "vuedraggable.min.js",
+  "description": "Vue component allowing drag-and-drop sorting in sync with View-Model. Based on Sortable.js.",
+  "keywords": [
+    "vue",
+    "vuejs",
+    "drag and drop",
```

Comment:
I don't think the keyword `and` makes sense simply on its own. Let's not use that?